docs/INSTALL.md

```diff
@@ -29,6 +29,18 @@ The code is available via github: https://github.com/OpenUpSA/pombola
 git clone https://github.com/OpenUpSA/pombola.git
 ```
 
+Run migrations
+
+```
+docker-compose run --rm app python manage.py migrate
+```
+
+Add dev data for easy dev setup
+
+```
+docker-compose run --rm app python manage.py loaddata demodata.json
+```
+
```

Comment:
this stuff belong after starting the db though, right? :)

Try going through the instructions from scratch and see if they actually work

Reply:
Kuda had tried it in this order and it worked fine